rmtk/vulnerability/derivation_fragility/NLTHA_on_SDOF/MSA_on_SDOF.py

```diff
@@ -61,8 +61,14 @@ def run_time_history_analysis(capacity_curves, hysteresis, icc,gmrs,target_name,
     igmr = gmrs['name'].index(target_name)
     write_gmr(gmrs,igmr, scaling_factor)
     create_SDOF_paramfile(capacity_curves, hysteresis, icc,damping,degradation)
-#    os.system('chmod +x OpenSees')
-    os.system('./OpenSees simplestSDOFgen.tcl')
+    try:
+        os.system('OpenSees simplestSDOFgen.tcl')
+    except:
+        pass
+    try:
+        os.system('./OpenSees simplestSDOFgen.tcl')
+    except:
+        pass
```

Comment:
```try:
    os.system(...)
```
will never raise an exception. Please use (at least) `subprocess.call()` instead and catch the `OSError` exception